src/controllers/showController.ts

```diff
@@ -1,40 +1,84 @@
+import { Show } from '../models';
+import { ShowAttributes, ShowInstance} from '../models/Show';
+
 /**
- * Creates a show with the given attribues
- *
- * @param attributes
- * @returns The newly created show.
- */
-export function createShow(attributes: any) {}
+* Creates a show with the given attribues
+*
+* @param {ShowAttributes} attributes
+* @returns The newly created show.
```

Comment:
Can each JSDoc comment have the following properties:
- A short description!
- [`param`](http://usejsdoc.org/tags-param.html)
- [`async`](http://usejsdoc.org/tags-async.html) if applicable
- [`returns`](http://usejsdoc.org/tags-returns.html)
- [`author`](http://usejsdoc.org/tags-author.html)